src/sentry/static/sentry/app/components/discover/transactionsList.tsx

```diff
@@ -0,0 +1,370 @@
+import React from 'react';
+import {Location, LocationDescriptor, Query} from 'history';
+import {browserHistory} from 'react-router';
+import styled from '@emotion/styled';
+
+import {Client} from 'app/api';
+import {t} from 'app/locale';
+import DiscoverButton from 'app/components/discoverButton';
+import DropdownControl, {DropdownItem} from 'app/components/dropdownControl';
+import SortLink from 'app/components/gridEditable/sortLink';
+import Link from 'app/components/links/link';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import Pagination from 'app/components/pagination';
+import PanelTable from 'app/components/panels/panelTable';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import {Organization} from 'app/types';
+import DiscoverQuery, {TableData, TableDataRow} from 'app/utils/discover/discoverQuery';
+import EventView, {MetaType} from 'app/utils/discover/eventView';
+import {Sort, getAggregateAlias} from 'app/utils/discover/fields';
+import {getFieldRenderer} from 'app/utils/discover/fieldRenderers';
+import {decodeScalar} from 'app/utils/queryString';
+import HeaderCell from 'app/views/eventsV2/table/headerCell';
+import {TableColumn} from 'app/views/eventsV2/table/types';
+import {GridCell, GridCellNumber} from 'app/views/performance/styles';
+
+const DEFAULT_TRANSACTION_LIMIT = 5;
+
+export type DropdownOption = {
+  /**
+   * The sort to apply to the eventView when this is selected.
+   */
+  sort: Sort;
+  /**
+   * The unique name to use for this option.
+   */
+  value: string;
```

Comment:
nit: without the comment, I wasn't sure what this did. Thoughts on renaming to `key` or similar since it has to be unique?

Reply:
I believe this naming originates from html options using the value attribute for this. Do you still think using `key` will be clearer here?

Reply:
Nah we can keep it, not a big deal either way :+1: